yt/yt/ytlib/query_tracker_client/helpers.cpp

```diff
@@ -17,23 +17,23 @@ TString GetFilterFactors(const TActiveQueryPartial& record)
     return Format("%v %v aco:%v",
         record.Query,
         (record.Annotations && *record.Annotations) ? ConvertToYsonString(*record.Annotations, EYsonFormat::Text).ToString() : "",
-        (record.AccessControlObject && *record.AccessControlObject) ? **record.AccessControlObject : "");
+        record.AccessControlObjects ? ConvertToYsonString(*record.AccessControlObjects, EYsonFormat::Text).ToString() : "[]");
```

Comment:
It would not be very convenient to search by aco.

Can we do something like this:
```
return Format("%v %v acos:%v",
        record.Query,
        (record.Annotations && *record.Annotations) ? ConvertToYsonString(*record.Annotations, EYsonFormat::Text).ToString() : "",
        record.AccessControlObjects ? ConvertToYsonString(AddPrefixToEachElement(*record.AccessControlObjects, "aco:"), EYsonFormat::Text).ToString() : "[]");
```

This way the produced string will contain a substring `acos:[aco:nobody, aco:everyone, aco:my-team]` and users can search correctly by `aco:my-team`(contains) and `acos:[aco:aco1, aco:aco2]`(consistsOf)

Search happens simply as substring